src/vs/workbench/api/browser/mainThreadTreeViews.ts

```diff
@@ -31,19 +32,20 @@ export class MainThreadTreeViews extends Disposable implements MainThreadTreeVie
 		this._proxy = extHostContext.getProxy(ExtHostContext.ExtHostTreeViews);
 	}
 
-	async $registerTreeViewDataProvider(treeViewId: string, options: { showCollapseAll: boolean, canSelectMany: boolean, canDragAndDrop: boolean }): Promise<void> {
+	async $registerTreeViewDataProvider(treeViewId: string, options: { showCollapseAll: boolean, canSelectMany: boolean, dragAndDropController?: DragAndDropController<any> }): Promise<void> {
```

Comment:
Can you use `ITreeViewDragAndDropController` here so that `vscode` doesn't need to be imported?